src/main/java/hudson/plugins/selenium/configuration/browser/webdriver/ChromeBrowser.java

```diff
@@ -30,7 +30,7 @@ public ChromeBrowser(int maxInstances, String version, String server_binary) {
     @Override
     public Map<String, String> getJVMArgs() {
         Map<String, String> args = new HashMap<String, String>();
-        combine(args, PARAM_BINARY_PATH, getServer_binary());
+        combine(args, paramBinaryPath, getServerBinary());
```

Comment:
When changing method names please keep in mind that jenkins makes use of a templating engine called jetty for the webui, which is linked to variables and methods through fieldnames.

For example the jetty ui file for the chrome browser is [this](https://github.com/jenkinsci/selenium-plugin/blob/master/src/main/resources/hudson/plugins/selenium/configuration/browser/webdriver/ChromeBrowser/config.jelly)

as you can see it references the field server_binary. You just changed that method, but not the reference in the ui file, thus the plugin can now no longer save the path to the chromedriver binary.

I started working on this plugin (or any jenkins plugin for that matter) only some time ago so I do not have a good grasp of the internals of the plugin, so this makes it somewhat of a risk to implement your PR's. That is not to say that I do not welcome the work you did, I am happy for it. But I am sure you can see why I am not completely happy with having to track down bugs you have created, when I myself are still wet behind the ears when it comes to this plugin, jetty and jenkins.

Also your pr's contained a number of merge conflicts which I had to manually fix. I have merged your changes locally and will push them when I have restored functionality in the webui and made sure nothing else broke.

Please do not take this too negative, I am just trying to let you know I am new to this but do appreciate it.